.github/ISSUE_TEMPLATE.md

```diff
@@ -16,4 +16,4 @@
 - Dedicated / Self-Hosted Multiplayer
 
 **RPT log file:**
-- Add a link ([gist](https://gist.github.com) or [pastebin](http://pastebin.com)) to the client and/or server RPT file. An instruction to find your RPT files can be found [here](https://community.bistudio.com/wiki/Crash_Files#Arma_3).
\ No newline at end of file
+- Add a link ([gist](https://gist.github.com) or [pastebin](http://pastebin.com)) to the client and/or server RPT file. An instruction to find your RPT files can be found [here](https://community.bistudio.com/wiki/Crash_Files#Arma_3).

```

Comment:
Github inside webbrowser seems to not be able to fix it... I'll do that tomorrow then...